app/src/main/res/values/strings.xml

```diff
@@ -184,21 +185,21 @@
     <string name="importVoucherVaultMessage">Select your <i>vouchervault.json</i> export from Voucher Vault to import.
 \nCreate it by pressing Export in Voucher Vault first.</string>
     <string name="barcodeId">Barcode value</string>
-    <string name="sameAsCardId">Same as card ID</string>
+    <string name="sameAsCardId">Same as ID</string>
     <string name="setBarcodeId">Set barcode value</string>
     <string name="unsupportedBarcodeType">This barcode type can\'t yet be displayed. It may be supported in a later version of the app.</string>
     <string name="wrongValueForBarcodeType">The value is not valid for the selected barcode type</string>
-    <string name="copy_to_clipboard_multiple_toast">Card IDs copied to clipboard</string>
+    <string name="copy_to_clipboard_multiple_toast">IDs copied to clipboard</string>
     <string name="intent_import_card_from_url_share_multiple_text">I want to share some cards with you</string>
-    <string name="frontImageDescription">Card\'s front image</string>
-    <string name="backImageDescription">Card\'s back image</string>
+    <string name="frontImageDescription">Frontside picture</string>
+    <string name="backImageDescription">Backside picture</string>
     <string name="photos">Photos</string>
     <string name="setFrontImage">Set front image</string>
-    <string name="setBackImage">Set back image</string>
+    <string name="setBackImage">Set reverse side image</string>
```

Comment:
Found on https://en.wikipedia.org/wiki/Credit_card

Reply:
Very inconsistent now though, because just 2 strings above you call it "Backside picture". What is it, "backside" or "reverse side"?

Reply:
It is both actually. If the user doesn't understand either "image" or "front/reverse", the description has a different term for the same thing. The descriptions are consistent with the descriptions, and this is (hopefully) more technically correct.

Reply:
>If the user doesn't understand either "image" or "front/reverse", the description has a different term for the same thing

It really isn't helpful for that situation because only screenreaders will read frontImageDescription and backImageDescription.

Reply:
And what Reverse translation are you referring to? `string name="reverse"`? Because that is the sort screen:
![image](https://user-images.githubusercontent.com/1885159/144718127-f52a9927-11d8-42ec-bd8d-a89f14dd2bf1.png)

But hearing you name "Invert" now, that would have been a more logical translation...

Reply:
I confused myself. Changed to "Reversed order" which works better with "sort by".
Could be "in reversed order" but I am not sure that works in all languages.

Reply:
Front vs Front-side
Reverse side vs Reverse-side

Please pick something for consistency's sake.

In fact, why not just "Front image" and "Back image". It's nice and simple and it won't take 3 tries to get it right.

Reply:
Sorry. I believe "Reverse-side image" to be correct, so went with that.